zkevm-circuits/src/super_circuit.rs

```diff
@@ -176,9 +184,13 @@ impl<F: Field, const MAX_TXS: usize, const MAX_CALLDATA: usize> Circuit<F>
             .evm_circuit
             .load_fixed_table(&mut layouter, self.fixed_table_tags.clone())?;
         config.evm_circuit.load_byte_table(&mut layouter)?;
-        config
-            .rw_table
-            .load(&mut layouter, &self.block.rws, self.block.randomness)?;
+        config.rw_table.load(
+            &mut layouter,
+            &self.block.rws.table_assignments(),
+            self.block.randomness,
+            self.block.state_circuit_pad_to,
```

Comment:
where is `state_circuit_pad_to` assigned ?

Reply:
it is left to do in the final circuit (aggregation circuit  / super circuit / etc) in the future. Currently we can either set it to (1<<k - 64), or in tests set it to 0 to indicate no padding needed.